docs/src/content/docs/ko/guides/sidebar.mdx

```diff
@@ -32,17 +32,15 @@ import SidebarPreview from '~/components/sidebar-preview.astro';
 <SidebarPreview
 	config={[
 		{
-			label: 'guides',
+			label: '별자리',
```

Comment:
```suggestion
			label: 'constellations',
```

The default sidebar will use the folder names and as they are not translated in this case, this would be just `constellations`.

Reply:
Thank you!